.travis.yml

```diff
@@ -7,17 +7,6 @@ os: osx
 osx_image: xcode6.4
 
 env:
-  matrix:
-    
-    - CONDA_NPY=111  CONDA_PY=27
-    - CONDA_NPY=112  CONDA_PY=27
-    - CONDA_NPY=113  CONDA_PY=27
-    - CONDA_NPY=111  CONDA_PY=35
-    - CONDA_NPY=112  CONDA_PY=35
-    - CONDA_NPY=113  CONDA_PY=35
-    - CONDA_NPY=111  CONDA_PY=36
-    - CONDA_NPY=112  CONDA_PY=36
-    - CONDA_NPY=113  CONDA_PY=36
```

Comment:
Looks like we have the same issue here. Would just manually revert the changes to `.travis.yml` and `appveyor.yml`. They are both caused by `libgpuarray` 0.7.0 missing packages for macOS and Windows respectively.

Reply:
I just did that